src/components/Contact/index.js

```diff
@@ -0,0 +1,3 @@
+import Contact from './Contact';
```

Comment:
Again, simple not reused things should not be passed to a separate component. You can just create an object - make it an immutable Map - in constants and then import and map it in a separate renderContacts method in sidebar.